internal/client/model.go

```diff
@@ -367,21 +367,23 @@ type EncryptedSecret struct {
 
 // create secrets.
 type CreateSecretV3Request struct {
```

Comment:
Since we are not supporting reminders for end to end encrypted (e2ee) endpoints, you can remove these new fields you added